crates/relayer/src/keyring/signing_keys.rs

```diff
@@ -0,0 +1,43 @@
+use core::any::Any;
+
+use hdpath::StandardHDPath;
+use serde::{de::DeserializeOwned, Serialize};
+
+use super::{errors::Error, KeyFile};
+use crate::config::AddressType;
+
+// /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\
+// WARNING: Changing this struct in backward incompatible way
```

Comment:
Doesn't comply with the trait

Reply:
Moved to `Secp256k1KeyPair` per a different comment.